dds/DCPS/transport/framework/TransportReactorTask.cpp

```diff
@@ -43,6 +43,7 @@ OpenDDS::DCPS::TransportReactorTask::TransportReactorTask(bool useAsyncSend)
 
   this->reactor_ = new ACE_Reactor(new ACE_Select_Reactor, 1);
   this->proactor_ = 0;
+  this->reactor_owner_ = ACE_OS::NULL_thread;
```

Comment:
This should be moved to the initializer list so that in the case of useAsyncSend it is also initialized.